### PR TITLE
Icons and More

### PR DESCRIPTION
**Pull request by sri2k1us**
- Update Data and Analyses icons in `Search`, `Sharing` and `Bag`.
- Add more delineation to dashboard sections.
- 
image

image
image
image

### CODE REVIEW

**Review by ianmcorvidae — APPROVED** [review]
Looks reasonable to me. I don't know if other UI folks will have stronger opinions about some of the code, but the icon stuff seems good.

File: src/components/dashboard/DashboardSection.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import React, { useState } from "react";
 import clsx from "clsx";
 
-import { Typography, Collapse, Button } from "@material-ui/core";
+import {
+    Typography,
+    Paper,
+    Collapse,
```
Comment (ash3rz) [inline src/components/dashboard/DashboardSection.js:7]: GitHub Actions complaining about `Paper` being defined but not used

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

File: src/components/dashboard/DashboardSection.js (hunk 1 above)
Comment (ash3rz) [reply src/components/dashboard/DashboardSection.js:7]: Oh, also there's a merge conflict I forgot to mention

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**